cli/pcluster/config/mappings.py

```diff
@@ -538,10 +545,12 @@
                 "validators": [ec2_iam_role_validator],  # TODO add regex
             }),
             ("s3_read_resource", {
-                "cfn_param_mapping": "S3ReadResource",  # TODO add validator
+                "cfn_param_mapping": "S3ReadResource",
+                "validators": [s3_bucket_validator],
```

Comment:
s3_read_resource and s3_read_write_resource are arns pointing to a bucket or bucket objects and cannot be validated by s3_bucket_validator

Reply:
Good catch!